src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamNetworkStreamTest.cs

```diff
@@ -410,7 +410,7 @@ public async Task SslStream_NegotiateClientCertificateAsync_ServerDontDrainClien
         [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.SupportsTls13))]
         [InlineData(true)]
         [InlineData(false)]
-        [PlatformSpecific(TestPlatforms.Windows)]
+        [PlatformSpecific(TestPlatforms.Windows | TestPlatforms.Linux)]
```

Comment:
I think at this point SupportsTls13 should be sufficient. e.g. we will have support on all platforms supporting Tls 1.3